changelogs/unreleased/3536-multi-digit-python-version.yml

```
@@ -0,0 +1,5 @@
+change-type: patch
+issue-nr: 3536
+description: Fixes a bug where minor version of python were wrongly set as only the first digit of multi-digit number was used.
+destination-branches: [iso4]
+  bugfix: "minor version of python were wrongly set"
```

* The `sections` element was removed.
* You can always use the `{{description}}` placeholder in any of the sections. This placeholder will be replaced with the content of the `description` element.
```suggestion
change-type: patch
issue-nr: 3536
description: Fixes a bug where minor version of python were wrongly set as only the first digit of multi-digit number was used.
destination-branches: [iso4]
sections:
  bugfix: "{{description}}"
```